ArduCopter/Copter.cpp

```diff
@@ -548,11 +559,11 @@ void Copter::ten_hz_logging_loop()
 // twentyfive_hz_logging - should be run at 25hz
 void Copter::twentyfive_hz_logging()
 {
-    if (should_log(MASK_LOG_ATTITUDE_FAST)) {
+    if (should_log(MASK_LOG_ATTITUDE_FAST) && (!should_log(MASK_LOG_ATTITUDE_MED))) {
         Log_Write_EKF_POS();
     }
 
-    if (should_log(MASK_LOG_IMU)) {
+    if (should_log(MASK_LOG_IMU) && !(should_log(MASK_LOG_IMU_FAST))) {
```

Comment:
```suggestion
    if (should_log(MASK_LOG_IMU_FAST)) {
```
I think the 10 hz logging should check or not fast.

Reply:
IMU is 10 and FAST IMU is loop rate...there was no FAST IMU logging before this...